Sources/MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -336,40 +326,36 @@ open class NavigationEventsManager {
     }
     
     /**
-     Send feedback about the current road segment/maneuver to the Mapbox data team.
+     Create feedback about the current road segment/maneuver to be sent to the Mapbox data team.
      
      You can pair this with a custom feedback UI in your app to flag problems during navigation such as road closures, incorrect instructions, etc.
      
-     @param type A `FeedbackType` used to specify the type of feedback
-     @param description A custom string used to describe the problem in detail.
-     @return Returns a UUID used to identify the feedback event
+     @return Returns a feedback event
      
      If you provide a custom feedback UI that lets users elaborate on an issue, you should call this before you show the custom UI so the location and timestamp are more accurate.
      
-     You can then call `updateFeedback(uuid:type:source:description:)` with the returned feedback UUID to attach any additional metadata to the feedback.
+     You can then call `sendFeedback(_:type:source:description:)` with the returned feedback to attach any additional metadata to the feedback and send it.
```

Comment:
This requirement should be in a `postcondition` callout so that developers don’t miss it.